pkgs/misc/vim-plugins/update.py

```diff
@@ -442,19 +437,6 @@ def rewrite_input(input_file: Path, output_file: Path, redirects: dict):
         with open(DEPRECATED, "w") as f:
             json.dump(deprecations, f, indent=4, sort_keys=True)
 
-        print(
-            f"""\
-Redirects have been detected and {input_file} has been updated. Please take the
-following steps:
-    1. Run this script again so these changes will be reflected in the
-    generated expressions:
-            ./update.py
-    2. Commit {input_file} along with deprecations and generated expressions:
-            git add {output_file} {input_file} {DEPRECATED}
-            git commit -m "vimPlugins: Update redirects"
-        """
-        )
-
```

Comment:
If we're removing these instructions its even more reason to default to `--commit` (I'm reviewing the commits in order, in case some of my comments are resolved by future commits).